### PR TITLE
Removed html tag and fixed typo

### PR DESCRIPTION
**Pull request by klee97**

### CODE REVIEW

File: docs/testsuite_overview.md [hunk 1]
```diff
@@ -4,15 +4,16 @@ title: Test Suite
...
-Rest.li Client Cross-Language Test Suite
+Rest.li Cross-Language Test Suite Framework
 =========================
 
-The Rest.li Client Cross-Language Test Suite is a framework for testing and comparing Rest.li client implementations.
-It contains the Rest.li Test Suite Specification, which provides test data
+The Rest.li Cross-Language Test Suite is a framework for testing and comparing Rest.li implementations.
+The framework contains the Rest.li Test Suite Specification, which provides test data
 and testing guidelines that specify which features and scenarios to test, and how to test them.
 These test guidelines should be followed when testing a particular language implementation of Rest.li client.
```
Comment (mchen07) [inline docs/testsuite_overview.md]: remove "client".

File: docs/testsuite_overview.md [hunk 2]
```diff
@@ -266,9 +264,10 @@ the correct values. We compare the response's status and error message with the
 Next Steps
 ------------------
 ### Improvements to Test Suite Specification
-* add test for record with lowercase name 
-* add test for tunneled query params
-* add test that includes unicode (non-ascii) characters on wire
+* Add test for record with lowercase name 
+* Add test for tunneled query params
+* Add test that includes unicode (non-ascii) characters on wire
+* Extend spec to include server-side testing
```
Comment (mchen07) [inline docs/testsuite_overview.md]: include cross-language server testing.

**Comment by maximelamure** [comment]
LGTM
